pubsub/azure/eventhubs/eventhubs.go

```diff
@@ -157,7 +158,8 @@ func (aeh *AzureEventHubs) Init(metadata pubsub.Metadata) error {
 		return err
 	}
 	aeh.metadata = m
-	hub, err := eventhub.NewHubFromConnectionString(aeh.metadata.connectionString)
+	hub, err := eventhub.NewHubFromConnectionString(aeh.metadata.connectionString,
```

Comment:
Do we want to include version too? Maybe follow the `<product>/<version>` pattern, similar as in https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
Yes we do -- but how can I get the version? :D This is in components contrib after all.

Reply:
@artursouza also this isn't a HTTP header, so other user agents don't apply. The way these SDKs are implemented what is sent will be `$ourstring/$theirstring`

Reply:
We don't need to match the HTTP header but it is just as inspiration. The version is important. We already set the version in the dapr build, we can add the version to another public property too, this time in components contrib. Take a look at the Makefile in dapr/dapr repo.